bitnami/spring-cloud-dataflow/README.md

```diff
@@ -322,6 +322,7 @@ helm uninstall my-release
 | `mariadb.auth.forcePassword`              | Force users to specify required passwords in the database                                           | `false`      |
 | `mariadb.auth.usePasswordFiles`           | Mount credentials as a file instead of using an environment variable                                | `false`      |
 | `mariadb.initdbScripts`                   | Specify dictionary of scripts to be run at first boot                                               | `{}`         |
+| `flyway.enabled`                          | Enable/disable flyway running Dataflow and Skipper Database creation scripts on startup             | `true`       |
```

Comment:
If I'm not wrong, the default behavior (the one used now), is the equivalent to having this parameter set to `false`. I think it's safer to use `false` as the default value and allow users to modify it when it's needed.

Reply:
I am thinking with respect of first time users using helm install Spring-Cloud-Dataflow, who have not overridden values.yaml.
I think it is better to create schema on MariaDB StatefulSet, when server starts up.
Only when user starts using pre-created database, flyway.enabled can then be set as `false` in values override yaml file.

Reply:
Thanks, I will check the upgrade from previous chart versions (`4.0.X` -> `4.1.0`); if everything works fine, the PR will be merged.